src/shared/debug_op.c

```diff
@@ -575,3 +575,35 @@ char * win_strerror(unsigned long error) {
     return messageBuffer;
 }
 #endif
+
+void mt_log_wrapper(const char* log_type, const char *tag, const char * file, int line, const char * func, const char *msg, ...){
```

Comment:
I created a wrapper so `rsync` can call any type of log function (info, error, etc.) but passing only one.
To identify the type of log, I used a string because using an enumeration could:
- Create a dependency in the compilation of `rsync` by having to include that definition
- Or duplicate the code definition in both sides

I assumed the code penalty of the string comparison to avoid this situations and to make the code more clear.

But I'll make the modification. 
Thank you.